source/slang/check.cpp

```diff
@@ -8947,13 +8898,13 @@ namespace Slang
                 // For now let's just be expedient and disallow all of that, because
                 // we can always add it back in later.
 
-                if(!lookupResult.isOverloaded())
+                if (!lookupResult.isOverloaded())
```

Comment:
This space here goes against the formatting I'd like us to use (although I know VS loves to inject it). This is another case where I'd like to keep unrelated formatting changes out of a PR if possible.